com.unity.shadergraph/Editor/Utilities/Unity.ShaderGraph.Utilities.asmdef

```diff
@@ -10,5 +10,11 @@
     "precompiledReferences": [],
     "autoReferenced": false,
     "defineConstraints": [],
-    "versionDefines": []
+    "versionDefines": [
+        {
+            "name": "com.unity.rendering.hybrid",
+            "expression": "0.14.0-preview.0",
+            "define": "ENABLE_HYBRID_RENDERER_V2"
```

Comment:
I assume from the way this is set up that this is only defined when the given package with the given version number is present?

Not an expert on the asmdef syntax, but if that is true, then this looks correct.  :)

Reply:
Yeah almost. It is meant to define `ENABLE_HYBRID_RENDERER_V2` when the hybrid renderer package version `0.14.0-preview.0` _or newer_ is present. So it should apply to 0.15 and etc for example. 

In case you want the sources, it's from the Unity doc :

```
As a shortcut, you can enter a single version without range brackets to indicate that the expression includes that version or later:

2.1.0-preview.7 evaluates to x >= 2.1.0-preview.7
```

https://docs.unity3d.com/2021.2/Documentation/Manual/ScriptCompilationAssemblyDefinitionFiles.html